frontend_build/src/webpack.config.base.js

```diff
@@ -75,6 +75,7 @@ var config = {
         test: /\.vue$/,
         loader: 'vue-loader',
         options: {
+          preserveWhitespace: false,
```

Comment:
Whitespace between inline-blocks was the issue. Notice the difference below. I just realized this caused regressions for buttons side by side that lack margins. Will address those soon.

## Before

![before 1](https://user-images.githubusercontent.com/7193975/27337938-b78a3fb2-5588-11e7-8a39-d2f5b6a9dc2d.png)

## After

![after 1](https://user-images.githubusercontent.com/7193975/27337939-b7a1a404-5588-11e7-9e25-5c2a546c7dc9.png)